python/orca/src/bigdl/orca/learn/tf2/pyspark_estimator.py

```diff
@@ -635,8 +643,15 @@ def get_model(self,
         else:
             model = load_model(**self.load_params)
 
-        if set_weights and self.model_weights is not None:
-            model.set_weights(self.model_weights)
+        if set_weights:
+            if self.model_weights is not None:
+                model.set_weights(self.model_weights)
+            if self.optimizer_weights is not None:
+                import tensorflow as tf
+                grad_vars = model.trainable_weights
+                zero_grads = [tf.zeros_like(w) for w in grad_vars]
+                model.optimizer.apply_gradients(zip(zero_grads, grad_vars))
+                model.optimizer.set_weights(self.optimizer_weights)
```

Comment:
shall we set optimizer weights before setting model weights?

Referred to this post: https://stackoverflow.com/questions/49503748/save-and-load-model-optimizer-state
![image](https://user-images.githubusercontent.com/21994046/220033769-025eb7b8-11ad-4b9d-89b8-d72eeb86aef2.png)